docs/streaming-programming-guide.md

```diff
@@ -1626,10 +1626,10 @@ See the full [source code]({{site.SPARK_GITHUB_URL}}/blob/v{{site.SPARK_VERSION_
 # Lazily instantiated global instance of SparkSession
 def getSparkSessionInstance(sparkConf):
     if ("sparkSessionSingletonInstance" not in globals()):
-        globals()["sparkSessionSingletonInstance"] = SparkSession \
-            .builder \
-            .config(conf=sparkConf) \
-            .getOrCreate()
+        globals()["sparkSessionSingletonInstance"] = (SparkSession
```

Comment:
Maybe I am wrong. Could you maybe provide the reference? 

> recommended by pep8

Do you refer this line?

>The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, 
>brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. 
>These should be used in preference to using a backslash for line continuation.

I know the rule with binary operator follows this but I guess this case is not disallowed. I am not sure if it is worth sweeping all. They look preferred but not breaking pep8.